R/read_hivproj_param.R

```diff
@@ -74,10 +94,19 @@ read_hivproj_param <- function(pjnz_path) {
     cd4_prog = progress_cd4,
     cd4_mort = mortality_cd4,
     art_mort = art_mortality_cd4,
-    artmx_timerr = art_mortality_rates
+    artmx_timerr = art_mortality_rates,
```

Comment:
This section also looks a lot like data

Reply:
See [above comment](https://github.com/mrc-ide/specio/pull/1#discussion_r)